src/registrar/assets/js/get-gov.js

```diff
@@ -1702,6 +1768,7 @@ document.addEventListener('DOMContentLoaded', function() {
               <td data-sort-value="${new Date(request.last_submitted_date).getTime()}" data-label="Date submitted">
                 ${submissionDate}
               </td>
+              ${markupCreatorRow}
```

Comment:
this is a possible nitpick but is there any way to bring the (portfolioVal) closer this reference?

Reply:
yes I am referring to that! I thought having it closer would make it more readable. I think this might be just a nitpick thing. I understand what you are saying @CocoByte! So I will resolve